04_numeronym/alanhillgemann/main.go

```diff
@@ -0,0 +1,40 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+)
+
+var out io.Writer = os.Stdout
+
+func main() {
+	fmt.Fprint(out, numeronyms("accessibility", "Kubernetes", "abc"))
+}
+
+func numeronyms(vals ...string) []string {
+
+	numeronyms := make([]string, 0)
+	for _, val := range vals {
+		if val == "" {
+			numeronyms = append(numeronyms, "")
+		} else {
+			strArr := []rune(val)
```

Comment:
Name could be more descriptive. Maybe `runes`?

Reply:
Fixed. All variables renamed to be more descriptive